src/backends/pgfplotsx.jl

```diff
@@ -195,49 +179,47 @@ function (pgfx_plot::PGFPlotsXPlot)(plt::Plot{PGFPlotsXBackend})
                             "colorbar sampled" => nothing,
                         )
                     end
-                    # goto is needed to break out of col and series for
-                    @goto colorbar_end
+                    break
                 end
             end
-            @label colorbar_end
 
             if hascolorbar(sp)
-                cticks = get_colorbar_ticks(sp)[2]
-                if sp[:colorbar] === :top
-                    colorbar_style = PGFPlotsX.Options("xlabel" => sp[:colorbar_title])
+                cticks = curly(join(get_colorbar_ticks(sp)[2], ','))
+                colorbar_style = if sp[:colorbar] === :top
                     push!(
-                        colorbar_style,
-                        "at" => string((0.5, 1.05)),
+                        Options("xlabel" => sp[:colorbar_title]),
+                        "at" => "(0.5, 1.05)",
                         "anchor" => "south",
-                        "xtick" => string("{", join(cticks, ","), "}"),
+                        "xtick" => cticks,
                         "xticklabel pos" => "upper",
                         "xticklabel style" => pgfx_get_colorbar_ticklabel_style(sp),
                     )
                 else
-                    colorbar_style = PGFPlotsX.Options("ylabel" => sp[:colorbar_title])
                     push!(
-                        colorbar_style,
-                        "ytick" => string("{", join(cticks, ","), "}"),
+                        Options("ylabel" => sp[:colorbar_title]),
+                        "ytick" => cticks,
                         "yticklabel style" => pgfx_get_colorbar_ticklabel_style(sp),
                     )
                 end
                 push!(
                     axis_opt,
-                    string("colorbar", pgfx_get_colorbar_pos(sp[:colorbar])) => nothing,
```

Comment:
I disagree, especially with multiple arguments, the string interpolation is more readable and emphases the string content instead of the individual elements. And you save a consequent amount of characters, which by definition of concision is more readable.